src/components/Callout/Callout.Props.ts

```diff
@@ -59,6 +59,14 @@ export interface ICalloutProps extends React.Props<Callout>, IPositionProps {
    * If true do not render on a new layer. If false render on a new layer.
    */
   doNotLayer?: boolean;
+
+  /**
+   * If true then the callout will attempt to focus the first focusable element that it contains.
```

Comment:
If it doesn't find a focusable element, no focus will be set and the method will return false.
@returns True if focus was able to be set, false otherwise.